src/components/DeletionConfirmation/DeletionConfirmation.tsx

```diff
@@ -1,14 +1,34 @@
 import Button from '../Button/Button'
-import CloseButton from '../CloseButton/CloseButton'
 import style from './Deletionconfirmation.module.scss'
+import { useContext } from 'react'
+import { Context } from '../Context/Context'
+import { ContextData } from '../Context/ContextData'
 
-const DeletionConfirmation = () => (
-  <div className={style.deletion_confirmation}>
-    <CloseButton />
-    <h2>Delete MOVIE</h2>
-    <p>Are you sure you want to delete this movie? </p>
-    <Button title={'confirm'} />
-  </div>
-)
+interface deleteMovieProps {
+  onClick: () => void
+}
 
+
+const DeletionConfirmation = ({ onClick }: deleteMovieProps) => {
+  const value = useContext(Context)
+  const valueData = useContext(ContextData)
```

Comment:
you should pick better naming, `value` is too generic. Consider choosing less generic names like `movie`, `movieData` etc.